src/DynamoCoreWpf/Views/About/AboutWindow.xaml

```diff
@@ -62,7 +62,7 @@
                                PreviewMouseDown="OnPreviewMouseDown"
                                HorizontalAlignment="Center">
                         <Run Foreground="{Binding IsUpdateAvailable, Converter={StaticResource IsUpdateAvailableBrushConverter}}" 
-                             Text="{Binding IsUpdateAvailable, Converter={StaticResource IsUpdateAvailableToTextConverter}, 
+                             Text="{Binding Model.UpdateManager, Converter={StaticResource IsUpdateAvailableToTextConverter}, 
```

Comment:
Can we not bind this to the `IsUpdateAvailable` property on `UpdateManager` directly? Why do we need the converter?